.goreleaser.yml

```diff
@@ -213,3 +213,12 @@ release:
 
   extra_files:
     - glob: "./release/release-cosign.pub"
+
+rigs:
+  - rig:
+      owner: sigstore
+      name: fish-food
+    homepage: https://sigstore.dev
+    description: Container Signing, Verification and Storage in an OCI registry.
+    license: "Apache License 2.0"
```

Comment:
@dirien can you add the commit author to use the sigstore bot

```
commit_author:
  name: sigstore-bot
  email: 86837369+sigstore-bot@users.noreply.github.com
```

after that we are good to go

Reply:
Done :)